### DISCUSSION TITLE
How to wrap the dialog as a child component?

### DISCUSSION THREAD

**Discussion by chentao169** [quasarframework/quasar#13977] [CLI - PWA mode] [answered]
I wanna wrap the dialog as a child component and use a button in parent component to control if the dialog is displayed or not.
I found examples in https://quasar.dev/vue-components/dialog#qdialog-api use `v-model` to control this. 
However, if I pass `show` from parent to child component as a prop, I got error like `Unexpected mutation of "props" prop`.
So, what's the correct way to trigger a dialog as child component? thanks.

**Comment by MilosPaunovic** [comment]
See if one of this helps:
https://github.com/quasarframework/quasar/discussions?discussions_q=dialog+as+a+child

**Comment by chentao169** [comment, marked as answer]
https://stackoverflow.com/a/72488822/1230559
here is what I was looking for.
